contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -253,4 +256,34 @@ public final void testWritePrivateProfileSection() throws IOException {
             reader.close();
         }        
     }
+    
+    public void testGetResource() {
+		// On Windows 7, "14" is the type assigned to the "My Computer" icon
+		// (which is named "ICO_MYCOMPUTER")
+    	byte[] results = Kernel32Util.getResource("c:\\windows\\explorer.exe", "14", "ICO_MYCOMPUTER");
+    	
+    	assertNotNull(results);
+    	assertTrue(results.length == 272);
```

Comment:
Btw, if you're looking for examples or inspiration of tests for Win32 resources, check out https://github.com/dblock/resourcelib.